ui/components/App.vue

```diff
@@ -1,9 +1,32 @@
 <template>
   <div>
     <div class="uk-container uk-padding">
-      <h1>
-        Accounts
-      </h1>
+      <h1 v-text="$gettext('Accounts')" />
+      <oc-grid v-if="selectedAccountsAmount > 0" key="selected-accounts-info" gutter="small" class="uk-flex-middle">
+        <span v-text="selectionInfoText" />
+        <div>
+          <oc-action-drop class="accounts-actions-dropdown">
+            <template v-slot:button>
+              <span class="uk-margin-xsmall-right" v-text="$gettext('Actions')" />
+              <oc-icon name="expand_more" />
```

Comment:
Can we flip the icon into the other direction (= use the `expand_less` icon) when the drop is expanded? Only if it's easy to achieve.